imod/mf6/hfb.py

```diff
@@ -472,13 +494,20 @@ def render(self, directory, pkgname, globaltimes, binary):
             package, first convert to a Modflow6 package by calling pkg.to_mf6_pkg()"""
         )
 
+    def to_netcdf(self, *args, **kwargs):
+        new = deepcopy(self)
+        # kwargs.update({"encoding": self._netcdf_encoding()})
```

Comment:
I think you can remove this commented out code?

Reply:
Removed